modules/ducktests/tests/ignitetest/services/utils/templates/ignite.xml.j2

```diff
@@ -40,6 +40,12 @@
         <property name="failureDetectionTimeout" value="{{ config.failure_detection_timeout }}"/>
         <property name="systemWorkerBlockedTimeout" value="{{ config.sys_worker_blocked_timeout }}"/>
 
+        {% if config.version >= '2.8.0' %}
+            <property name="metricExporterSpi">
```

Comment:
should this be set only when tests require this explicitly?

Reply:
fixed

Reply:
fixed